style.css

```diff
@@ -0,0 +1,306 @@
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  background-image: url(img/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
+.subcontent {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+ul li {
+  margin-right: 10px;
+}
+
+.Union {
+  position: absolute;
+  left: 322px;
+  top: 27px;
+  border: 3px;
+}
+
+h2 {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  color: #172b4d;
+}
+
+h3 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+h3 span {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #7a869a;
+}
+
+h5 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  color: #172b4d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 30px auto;
+}
+
+.canopy-card {
+  background: #f7f7f9;
+  padding: 20px 0;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-gap: 1rem;
+}
+
+.snapshoot-portofolio {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.canopy-card-section {
+  display: flex;
+  background: #fff;
+  height: 580px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-second-card-section {
+  background: #fff;
+  height: 610px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-third-card-section {
+  background: #fff;
+  height: 573px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-fourth-card-section {
+  background: #fff;
+  height: 610px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.project-card {
+  width: 100%;
+  margin: -5px auto;
+}
+
+.project-card p {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+.project-card li {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03;
+  color: #6070ff;
+  background: #ebebff;
+  margin-right: 5px;
+  padding: 7px;
+  list-style: none;
+  border-radius: 8px;
+}
+
+.project-card ul {
+  padding: 0;
+}
+
+button {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03;
+  color: #396df2;
+  border: 1px solid #396df2;
+  padding-right: 30px;
+  margin: 16px auto;
+  border-radius: 8px;
+  background: #fff;
+  cursor: pointer;
+  display: block;
+  position: absolute;
+}
```

Comment:
- [ ] Kindly make sure to implement the `button` interaction like `Active, Hover and Pressed`.

  - ### FIGME DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219610396-4421ab7c-b4e4-43de-95be-89b9b946fee1.png)